paddle/scripts/paddle_build.sh

```diff
@@ -531,16 +531,17 @@ function assert_api_spec_approvals() {
     fi
 
     pip install ${PADDLE_ROOT}/build/opt/paddle/share/wheels/*.whl
-    CHECK_DOCK_MD5=`python ${PADDLE_ROOT}/tools/check_doc_approval.py`
-    if [ "True" != ${CHECK_DOCK_MD5} ]; then
+    python ${PADDLE_ROOT}/tools/check_doc_approval.py > new_md5.spec
+    sed -i 's#),#)\n#g' new_md5.spec
+    check_fluid_document=`python ${PADDLE_ROOT}/tools/diff_api.py ${PADDLE_ROOT}/paddle/fluid/fluid_document.spec new_md5.spec` 
```

Comment:
do we need to update fluid_document.spec in the PR like API.spec? How to do that?

Reply:
I only thought of this method, and I tried to make it the same as api. spec.